src/styles/css/main.css

```
@@ -0,0 +1,345 @@
+        margin: 0;
+    }
+    .navbar__button {
+        margin: 0;
+        width: auto;
+        background-color: transparent;
+        gap: 0;
+        z-index: 4;
+    }
+}
+body {
+    font-family: 'Roboto';
```

Biome (2.1.2)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)